tuning/trainercontroller/controllermetrics/history_based_metrics.py

```diff
@@ -21,12 +21,11 @@
 
 # Third Party
 from transformers import TrainerState
-from transformers.utils import logging
 
 # Local
 from tuning.trainercontroller.controllermetrics.metricshandler import MetricHandler
 
-logger = logging.get_logger(__name__)
+# Configure log level
```

Comment:
remove comment, no longer relevant

Reply:
@anhuong  Thanks for the quick review. I have made all changes mentioned above and pushed it again. Thank you!
As discussed, I can also wait for Dushyant's comment!